deploy/helm/e2e-extras/templates/registry-deployment.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
```

Comment:
did you mean to commit the empty .gitignore?

Reply:
Yes, we have it on the other module because helm expects that directory AFAIK, I think it was created by the helm initialization of the module.

Reply:
Just did it now with a 2nd commit :)